Automattic/wp-calypso
# Checkout: Remove old checkout's Checkout component

#### Changes proposed in this Pull Request

This removes the `Checkout` and `CheckoutContainer` components and all of their dependencies that were part of old checkout and are now unused.

Depends on https://github.com/Automattic/wp-calypso/pull/48501

#### Testing instructions

- Verify that (new) checkout still works and looks correct (it's vaguely possible that these stylesheets were somehow used there).
- Verify that neither `Checkout`, nor `CheckoutContainer` are still used anywhere.
